.github/workflows/java_tests.yml

```diff
@@ -164,6 +183,10 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: '1.16.12'
+      - name: Install Python
+        uses: actions/setup-python@v3
```

Comment:
Should we just be using our generic setup action? https://github.com/apache/beam/blob/master/.github/actions/setup-self-hosted-action/action.yml

Reply:
Right, it's a good chance to centralize installations for Self-Hosted, I've changed individual setups to composite for all jobs , will do the same for Python Tests and Build Wheels, thanks @damccorm.